client/components/Home.js

```diff
@@ -15,7 +15,7 @@ const Home = () => {
               <button>All Snacks</button>
             </Link>
             <Link to="/countries">
-              <button>All Countries</button>
+              <button>Add A Country</button>
```

Comment:
Hm this should not be on the home page. It should be a tab when you log in like the manage users and orders

Reply:
Ok,I will leave it to Russel